### ISSUE TITLE
recode-regression in 3.0.10

### ISSUE THREAD

**Issue by hurda** [jmacd/xdelta#205]
A commandline, which worked in previous versions, stopped working in 3.0.10:

```
xdelta3-3.0.9-x64.exe recode -v -S lzma test.xdelta test_lzma.xdelta
xdelta3: source <placeholder> source size 0 B [0] blksize 64.0 MiB window 64.0 MiB
xdelta3: 0: in 44.3 KiB: out 28.7 KiB: total in 44.3 KiB: out 28.7 KiB: 16 ms
xdelta3: 1: in 57.8 KiB: out 36.2 KiB: total in 102 KiB: out 65.0 KiB: 15 ms

...

xdelta3: 17: in 2.47 KiB: out 1.59 KiB: total in 1.03 MiB: out 670 KiB: 0 ms
xdelta3: finished in 203 ms; input 1078722 output 686812 bytes (63.67%)
```

```
xdelta3-x86_64-3.0.10.exe recode -v -S lzma test.xdelta test_lzma.xdelta
xdelta3: too many filenames: -S ...

```

**Comment by mgrinzPlayer** [comment]
jmacd compiled those with VCDIFF_TOOLS=0

**Comment by hurda** [comment]
> @all,
> If you want you can try those (binaries compiled on my machine, with VCDIFF_TOOLS=1)

Thank you for the compile!

**Comment by jmacd** [comment]
This sounds like an accident. If VCDIFF_TOOLS just works with mingw builds,
then there's no reason to leave it off. I'll make sure to fix this in the
next release 3.0.x (although I'm working on the 64bithash branch at
present).

**Comment by jmacd** [comment]
This is fixed at head: https://github.com/jmacd/xdelta-devel.git

**Comment by jmacd** [comment]
Fixed in 3.0.11
